staging/src/k8s.io/component-base/featuregate/testing/feature_gate.go

```diff
@@ -32,6 +32,20 @@ import (
 func SetFeatureGateDuringTest(tb testing.TB, gate featuregate.FeatureGate, f featuregate.Feature, value bool) func() {
 	originalValue := gate.Enabled(f)
 
+	// Specially handle AllAlpha and AllBeta
+	var cleanups []func()
+	if f == "AllAlpha" || f == "AllBeta" {
```

Comment:
The special features come from map at https://github.com/kubernetes/kubernetes/blob/964529b227331f1f7970aade54123413e795a267/staging/src/k8s.io/component-base/featuregate/feature_gate.go#L61-L64

does it make sense to use those map keys as source of truth for these values? In that case the `cleanup` list would likely need to reset ALL non-special features.

Reply:
I don't think I want to export that map or those keys just for this, and I think the cleanup should limit itself to resetting the state of the features that are actually modified by this call